docs/configuring.md

```diff
@@ -1146,43 +1146,51 @@ Like [`message_before_copy`][message_after_copy] but printed before
 
 ### `migrations`
 
--   Format: `List[dict]`
+-   Format: `List[str|List[str]|dict]`
 -   CLI flags: N/A
 -   Default value: `[]`
 
-Migrations are like [tasks][tasks], but each item in the list is a `dict` with these
-keys:
+Migrations are like [tasks][tasks], but each item can have additional keys:
 
--   **version**: Indicates the version that the template update has to go through to
-    trigger this migration. It is evaluated using [PEP 440][].
--   **before** (optional): Commands to execute before performing the update. The answers
-    file is reloaded after running migrations in this stage, to let you migrate answer
-    values.
--   **after** (optional): Commands to execute after performing the update.
+-   **command**: The migration command to run
+-   **version** (optional): Indicates the version that the template update has to go
+    through to trigger this migration. It is evaluated using [PEP 440][]. If not version
+    is specified the migration will run on every update.
```

Comment:
suggestion(docs): I think it's worth to mention that using ˋversionˋ can still save you from being marked as an unsafe template for every update, because the unsafe part will only happen in *that* specific update, and not in *any* update.

Reply:
Also "if not version" -> "if no version".

Reply:
Done.